python/pyarrow/_dataset.pyx

```diff
@@ -1899,7 +1912,7 @@ cdef shared_ptr[CScanContext] _build_scan_context(bint use_threads=True,
 
 cdef void _populate_builder(const shared_ptr[CScannerBuilder]& ptr,
                             list columns=None, Expression filter=None,
-                            int batch_size=32*2**10) except *:
+                            int batch_size=2**20) except *:
```

Comment:
Can this be put in a global variable (`DEFAULT_BATCH_SIZE`) rather than hard-coded?

Reply:
will do